static/js/compose.js

```diff
@@ -955,7 +955,7 @@ export function render_and_show_preview(preview_spinner, preview_content_box, co
     }
 
     if (content.length === 0) {
-        show_preview(i18n.t("Nothing to preview"));
+        show_preview($t({defaultMessage: "Nothing to preview"}));
```

Comment:
This gets inserted into the DOM and then cleaned as though it was previewed message body content; maybe worth scheduling a follow-up issue to be able to use `.text()` for this content just for readability.

Reply:
Ah yeah, this should be `$t_html` meanwhile.